app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id, dependent: :destroy
+  has_many :comments, foreign_key: :author_id, dependent: :destroy
+  has_many :likes, foreign_key: :author_id, dependent: :destroy
+
```

Comment:
![image](https://github.com/MMhaidari/blog-app/assets/93039359/caf0ba9f-b01c-47f3-abff-6e2f7975b8cd)

- [ ] It would be nice if you follow the above diagram and make your associations in a way to follow the project requirements 👍
- [ ] It would be great if you would have user_id in the comments and likes table as the foreign key of the users table instead of author_id because author_id should only exist in the post table 🤔 Kindly make this change to follow the above diagram 😊
- [ ] Kindly make this change in all the other models with respect to the above changes as well to make your app work as expected 👍